### ISSUE TITLE
Comark writes missing image alt texts as `undefined` strings

### ISSUE THREAD

**Issue by hendrikheil** [comarkdown/comark#146]
We just built a tiny internal syntax normalizer for nuxt-studio by using comarks parse & render APIs.

I noticed that comark is writing `undefined` into the alt texts of images that have nothing in the brackets in the source file.

```
![](/my/cool/path)
```
is turned into
```
![undefined](/my/cool/path)
```
after parsing and rendering back to markdown / mdc.

This screenshot is taken from our code after applying the normalizer we wrote:
<img width="2248" height="450" alt="Image" src="https://github.com/user-attachments/assets/4a75e5c9-48b7-4e07-bcec-1e96371cff50" />
